packages/components/molecules/f-mega-modal/README.md

```diff
@@ -54,54 +53,12 @@ export default {
         MegaModal: () => import(/* webpackChunkName: "mega-modal" */ '@justeat/f-mega-modal')
     }
 }
-
-```
-
-### Non-Vue Applications
-
-This module can be ran as a micro front-end for applications that don't make use of the Vue framework.
```

Comment:
Is there a plan to migrate this to a separate doc somewhere?

Reply:
Yes, it'll be put into the root doc.